Authenticator/Application/Support/Entitlements/Authenticator.entitlements

```diff
@@ -4,7 +4,8 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.com.bitwarden.authenticator</string>
+		<string>group.${BASE_BUNDLE_ID}</string>
+		<string>group.$(ORGANIZATION_IDENTIFIER).bitwarden-authenticator</string>
```

Comment:
⚠️ The group depends on whether the app is being built on PRD or Beta variant so I would extract the suffix `bitwarden-authenticator` to the xcconfig as well and update it accordingly in the `select_variant.sh` script. This also applies to the Info.plist file change.

Reply:
@fedemkr I've updated this PR as well with the same changes.